### ISSUE TITLE
RaspAP INSTALL ERROR: Unable to download files from github

### ISSUE THREAD

**Issue by janislejins** [billz/raspap-webgui#18]
having trouble using the installer script. 
I think there needs to be a .git at line 72... of common.sh install 
relevant section:
```
# Fetches latest files from github to webroot
function download_latest_files() {
    if [ ! -d "$webroot_dir" ]; then
        install_error "Web root directory doesn't exist"
    fi

    install_log "Cloning latest files from github"
    sudo git clone https://github.com/billz/raspap-webgui "$webroot_dir" || install_error "Unable to download files from github"
}
```

should be (?):
```
# Fetches latest files from github to webroot
function download_latest_files() {
    if [ ! -d "$webroot_dir" ]; then
        install_error "Web root directory doesn't exist"
    fi

    install_log "Cloning latest files from github"
    sudo git clone https://github.com/billz/raspap-webgui.git "$webroot_dir" || install_error "Unable to download files from github"
}
```

**Comment by janislejins** [comment]
made a fork with the change and still threw same error. Never mind I'll troubleshoot on my end.

**Comment by jrmhaig** [comment]
I haven't used the install scripts myself but I would guess that it is failing because `/var/www` (`$webroot_dir`) is not empty.

**Comment by billz** [comment]
Thanks for the heads up, I'll check it out.

**Comment by billz** [comment]
Fixed in #29.
